### ISSUE TITLE
Wiki, Q&A or Documentation

### ISSUE THREAD

**Issue by dagelf** [AnySoftKeyboard/AnySoftKeyboard#3047]
Is there a list of all the documentation somewhere in one place?

If not would it help if I created it?

**Comment by fretjex** [comment]
yes! and please let me know where you put it! :)

**Comment by menny** [comment]
This would be very helpful

**Comment by alchemiker** [comment]
I think I suggested and offered my help for a documentation a year ago or so and didn't get a response...

**Comment by dagelf** [comment]
Great! Let's do it. I just didn't want to spend time doing something if it was already existing.

**Comment by alchemiker** [comment]
@dagelf Yeah! When and where? :D And is there a chat-function or something similar in GitHub for discussing details?
